lib/js/jsconsole.nim

```diff
@@ -9,6 +9,19 @@
 
 ## Wrapper for the `console` object for the `JavaScript backend
 ## <backends.html#backends-the-javascript-target>`_.
+##
+## Styled Messages
+## ===============
+##
+## CSS-styled messages in the browser are useful for debugging purposes.
```

Comment:
unrelated to this PR, but PR welcome for adding to jsconsole console.dir

see https://developer.mozilla.org/en-US/docs/Web/API/Console/log#difference_between_log_and_dir

Reply:
Ye